_data/versions.yml

```diff
@@ -1355,17 +1403,17 @@ master:
   note: ""
   components:
      felix:
-      version: 2.x-series
+      version: 2.6.0-8-gecbd512d
       url: ""
      calicoctl:
       version: v1.x-series
       url: "https://www.projectcalico.org/builds/calicoctl"
       download_url: "https://www.projectcalico.org/builds/calicoctl"
      calico/node:
-      version: v2.x-series
+      version: v2.6.x-series
       url: ""
      calico/cni:
-      version: v1.11.0
+      version: v1.11.1
```

Comment:
shouldn't this stay at 1.11.0? https://github.com/projectcalico/cni-plugin/releases/tag/v1.11.0